wally.toml

```diff
@@ -5,5 +5,5 @@ registry = "https://github.com/UpliftGames/wally-index"
 realm = "shared"
 exclude = ["example", "docs", "Packages", ".moonwave", "build", "pages", "tests", "AttackOfTheKillerRoombas.rbxl", "AttackOfTheKillerRoombas.rbxl.lock", "matter.rbxl", "matter.rbxm", "roblox.toml"]
 
-[dependencies]
+[dev-dependencies]
 TestEZ = "roblox/testez@0.4.1"
```

Comment:
There's no newline at the end here. Can you add one? That should be enforced by our .editorconfig now as well, can you check that that's working for you or if I might have messed up the file?

Reply:
I have now pulled in the changes in main including the commits to .editorconfig, however that didn't seem to work properly? 

![image](https://github.com/evaera/matter/assets/68000848/d33f9f06-0b02-4e73-9f6a-0d0061823a5c)

Instead I will just add the newline manually.

Reply:
Do you have your extension configured correctly to use it? VS Code needs an extension since it doesn't support it natively.

Reply:
Oh wasn't aware of that, will check that out then